virtcontainers/qemu.go

```diff
@@ -1230,7 +1260,36 @@ func (q *qemu) addDevice(devInfo interface{}, devType deviceType) error {
 
 	switch v := devInfo.(type) {
 	case types.Volume:
-		q.qemuConfig.Devices = q.arch.append9PVolume(q.qemuConfig.Devices, v)
+		if q.config.VirtioFS {
+			q.Logger().Info("Adding vhost-user-fs-pci volume")
```

Comment:
Could you tweak this message (and the 9p one below) to use a structure format to make it more parseable - something like:

```go
q.Logger().WithField("volume-type", "vhost-user-fs-pci").Info("adding volume")
   :
q.Logger().WithField("volume-type", "virtio-9p").Info("adding volume")
```